tests/rhosak/kas_messaging.spec.ts

```diff
@@ -96,19 +96,24 @@ test.beforeEach(async ({ page }) => {
   expect(producerResponse === true).toBeTruthy();
 });
 
-test.afterAll(async ({ page }) => {
-  await navigateToKafkaList(page);
-
+test.afterEach(async ({ page }) => {
+  console.log('ENTRANDO EN EL AFTEREACH');
```

Comment:
remove all your logging info for your debugging

Reply:
removed. Can I resolve this conversation @kornys ?